src/styles/blooks/card.scss

```
@@ -0,0 +1,111 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: calc(200px - 34px);
+  width: 100%;
+  font-size: 12px;
+  line-height: 18px;
+  padding: 16px;
+  background-color: $white;
+  border-radius: 5px;
+  border: 1px solid $light-gray;
+  transition: transform .3s linear;
```

You repeat it 5 times, create variable for transition duration
```suggestion
  transition: transform ... linear;
```